src/core/model/schema.ts

```diff
@@ -301,6 +301,38 @@ export const buildJsonSchema = (element: SchemaElement) => {
   return result;
 };
 
+export const cleanLinkedElements = (schema: SchemaElement | undefined): any => {
+  if (!schema) {
+    return schema;
+  }
+  const clone = cloneDeep(schema) as any;
+  traverse(clone, (current: any) => {
```

Comment:
This `traverse` comes from a library which is meant to traverse actual JSON Schemas. I don't think we should use it here ;) However I think a `schemaElementTraverse` would be useful, so you could implement one and use that for cleaning. 
Alternatively I think you could just use lodash's `cloneDeepWith` which allows customizing the cloning. There you could just remove the linked elements.